epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -86,6 +94,28 @@ private void cancelAnimation(LottieAnimationView lottieView) {
   }
 
   @Override
+  public boolean canAnimateChanges(ColorHolder view) {
+    return false;
+  }
+
+  @Override
+  public Animator animateChanges(ColorHolder view) {
+    int currentColor = ((ColorDrawable) view.getBackground()).getColor();
```

Comment:
side note: it would be cool to use the old animator to gracefully update which color is being animated too, since if you toggle the "change" button quickly you can cycle through many colors.

Would also be cool to used the new Spring animation in the support library